src/main/java/roomescape/application/PendingPaymentService.java

```diff
@@ -0,0 +1,44 @@
+package roomescape.application;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import roomescape.application.request.PaymentInfo;
+import roomescape.domain.reservation.pendingpayment.PendingPayment;
+import roomescape.domain.reservation.pendingpayment.PendingPaymentRepository;
+import roomescape.domain.reservation.reserved.Reserved;
+import roomescape.domain.reservation.reserved.ReservedRepository;
+import roomescape.exception.NotFoundException;
+import roomescape.presentation.response.ReservedResponse;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class PendingPaymentService {
+
+    private final PaymentService paymentService;
+    private final ReservedRepository reservedRepository;
+    private final PendingPaymentRepository pendingPaymentRepository;
+
+    @Transactional
+    public ReservedResponse confirmPayment(final Long pendingPaymentId, final PaymentInfo paymentInfo) {
+        log.info("결제 요청 호출 - pendingPaymentId: {}, peymentKey= {}", pendingPaymentId, paymentInfo.paymentKey());
```

Comment:
```suggestion
        log.info("결제 요청 호출 - pendingPaymentId: {}, paymentKey= {}", pendingPaymentId, paymentInfo.paymentKey());
```

오타가 있네요.
info 로그는 어떤 기준으로 잡으셨는지 궁금해요

Reply:
`info`로그가 애플리케이션의 정상적인 동작을 나타내는 정보를 기록한다는 점이라고 생각하여, **서비스 코드 안에서 데이터가 바뀌는 경우**에 대해 `info`로그를 출력했습니다.

하지만 고민해보니, 위와 같이 결제 요청 메서드 호출에 대한 정보를 로그로 출력한다는 것은 정상적인 동작에 대한 정보 보다는 **실제 동작 흐름을 출력**하기 때문에 `debug`레벨에 해당하는 로그가 조금 더 적당하다고 생각하여, 수정하였습니다!

d6d2c53